#Mensagem de teste para o Tutorial de ROS do Toschi
#O objetivo da vida (e dessa mensagem) é ser feliz

int32 primeiroInteiro
#Inteiro de tipo primitivo

std_msgs/Int32 segundoInteiro
#Inteiro usando uma mensagem pré-pronta

string texto
#Texto de tipo primitivo

